tools/src/vendoring/legacy.ts

```diff
@@ -249,32 +288,50 @@ const vendoredModulesConfig: { [key: string]: VendoredModuleConfig } = {
     semverPrefix: '~',
     moduleModifier: GestureHandlerModifier,
     steps: [
+      {
+        sourceAndroidPath: 'android/src/main/java/com/swmansion/gesturehandler',
+        targetAndroidPath: 'modules/api/components/gesturehandler',
+        sourceAndroidPackage: 'com.swmansion.gesturehandler',
+        targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler',
+      },
       {
         sourceAndroidPath: 'android/lib/src/main/java/com/swmansion/gesturehandler',
         targetAndroidPath: 'modules/api/components/gesturehandler',
         sourceAndroidPackage: 'com.swmansion.gesturehandler',
         targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler',
+        cleanupTargetPath: false, // first step cleans parent directory
       },
       {
         sourceAndroidPath: 'android/common/src/main/java/com/swmansion/common',
         targetAndroidPath: 'modules/api/components/gesturehandler/common',
         sourceAndroidPackage: 'com.swmansion.common',
         targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler',
+        cleanupTargetPath: false, // first steps cleans parent directory
+      },
+      {
+        sourceAndroidPath: 'android/src/paper/java/com/facebook/react/viewmanagers',
+        targetAndroidPath: '../../../../com/facebook/react/viewmanagers',
+        sourceAndroidPackage: 'com.facebook.react.viewmanagers',
+        targetAndroidPackage: 'com.facebook.react.viewmanagers',
+        cleanupTargetPath: false,
+      },
+      {
+        sourceAndroidPath: 'android/src/paper/java/com/swmansion/gesturehandler',
+        targetAndroidPath: 'modules/api/components/gesturehandler',
+        sourceAndroidPackage: 'com.swmansion.gesturehandler',
+        targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler',
+        cleanupTargetPath: false,
       },
       {
         recursive: true,
         sourceIosPath: 'ios',
         targetIosPath: 'Api/Components/GestureHandler',
-        sourceAndroidPath: 'android/src/main/java/com/swmansion/gesturehandler/react',
-        targetAndroidPath: 'modules/api/components/gesturehandler/react',
-        sourceAndroidPackage: 'com.swmansion.gesturehandler',
-        targetAndroidPackage: 'versioned.host.exp.exponent.modules.api.components.gesturehandler',
       },
     ],
     warnings: [
-      `NOTE: Any files in ${chalk.magenta(
-        'com.facebook.react'
-      )} will not be updated -- you'll need to add these to expoview manually!`,
+      `NOTE: Any files in ${chalk.magenta('com.facebook.react')} will not be updated ${chalk.red(
+        'for iOS, Android is fine'
```

Comment:
could you share some insight that to mention ios here? for verdoring gesture-handler for ios, i think we don't use the legacy config here, but to use the new expoGoConfig.ts.